linkup-cli/src/local_server.rs

```diff
@@ -51,6 +52,116 @@ async fn linkup_config_handler(
     }
 }
 
+async fn linkup_ws_request_handler(
+    string_store: web::Data<MemoryStringStore>,
+    req: HttpRequest,
+    req_stream: web::Payload
+) -> impl Responder {
```

Comment:
This can probably be split a bit, but I guess that can be done at another time! 😄